probe-dictionary/explore.js

```diff
@@ -861,13 +861,20 @@ function showDetailViewForId(probeId, channel=$("#select_channel").val()) {
 
   // Recording range
   let rangeText = [];
+  let expiryText = [];
   for (let [ch, history] of Object.entries(probe.history)) {
     if ((!history[0].optout && (ch == "release")) || (ch == "aurora")) {
       continue;
     }
+
     rangeText.push(`${ch} ${friendlyRecordingRangeForHistory(history, ch, true)}`);
+
+    let expiry = history[0].expiry_version || "never";
+    let expireDetail = (expiry == "never") ? "never expires" : `stops recording in ${expiry}`;
```

Comment:
"stops recording in X" is present-tense, which means it'll read strangely for versions in the past like "stops recording in 42". We'll need a tense check based on `latestVersion` or some genius rewrite.

I'm sorry English is so bad at precisely stating negation with continuation.

Reply:
Can you suggest alternatives? I'm struggling to find a better, intuitive & precise description.

Reply:
If we stick with "stops recording in X" for present/future tense, we can use "stopped recording in X" for past.